ocs_ci/ocs/acm/acm.py

```
@@ -971,117 +971,193 @@ def get_clusters_env():
+                f"Failed to delete {kind} '{cluster_name}' during cleanup: {ex}"
+            )
+    log.info(f"Waiting for namespace '{cluster_name}' to be fully removed")
+    try:
+        OCP(kind="namespace").wait_for_delete(
+            resource_name=cluster_name, timeout=300, sleep=15
+        )
+        log.info(f"Namespace '{cluster_name}' fully removed")
+    except (CommandFailed, TimeoutError):
+        log.warning(
+            f"Namespace '{cluster_name}' not fully removed after 300s, "
+            f"proceeding with retry"
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)